### ISSUE TITLE
[Feature Request] add srcset support to VParallax

### ISSUE THREAD

**Issue by ram-you** [vuetifyjs/vuetify#1535]
Hi,
any plan for using v-parallax  with srcset and sizes ?
thanks

**Comment by johnleider** [comment]
When creating an issue you should be provided with boilerplate for the ticket. This is important for myself and others to properly address your issue. Please reopen this with the required information, thank you.

**Comment by ram-you** [comment]
@johnleider 
To manage an open source project, one must be kind and open-minded.
I noticed that your attitude is racist because whenever I propose a functionality, even if it is important or interesting, you automatically refuse it.

**Comment by johnleider** [comment]
That is because the issue board is for feature requests and bugs, not for questions. When creating an issue, you are supplied with boilerplate that says this:

Have a feature request?
=======================
- Remove the template from below and provide thoughtful commentary. 

- Answer those questions:
  - What will it allow you to do that you can't do today?
  - How will it make current work-arounds straightforward?
  - What potential bugs and edge cases does it help to avoid?

Do not create new features based on a problem that will only solve edge cases for your project.

Every time you create an issue, you are refusing to follow these guidelines, and because of that, the issues are closed. If you can take the time to fill out the proper requirements, I will be more than happy to consider your requests.

**Comment by ram-you** [comment]
> v-parallax with srcset, heights and sizes 
was not a question, it was a Feature-Request

**Comment by johnleider** [comment]
If it was a feature request, then you should follow the feature request boilerplate, provided above, and that exists when you go to create an issue.

**Comment by ram-you** [comment]
>  If it was a feature request, then you should follow the feature request boilerplate, provided above, and that exists when you go to create an issue.

A rule you apply firmly but not for everyone.
I know that you have understood my request but you refused it on the pretext that it does not respect the rules.
I was helping a lot in many open source projects for many years, but with Vuetify I'll just consume.
This does not prevent me from enjoying your project and respecting all the effort and quality of the work you do.

Thank you very much and sorry if I abused your time

**Comment by johnleider** [comment]
Can you just clarify to me what it is that you are wanting to add? We can take it from there. Thank you for your patience.

**Comment by ram-you** [comment]
**v-parallax**  have only  **src** and **height** options. After compilation generated an <img with the provided src and height .
I'd like to generate a **responsive image** with srcset, sizes & heights  so we should simply add srcset, heights and sizes as options.

Thank you again

**Comment by johnleider** [comment]
So essentially instead of applying an image to a background automatically for you, you'd like to be able to just use your own <img> tag within the carousel?

**Comment by ram-you** [comment]
in **VParallax.vue** src/components/VParallax I suggest this changes:

```
.....
    props: {
      height: {
        type: [String, Number],
        default: 500
      },
      src: String,
      srcset: String, 
      heights: String, 
      sizes: String
    },
......
    render (h) {
      const container = h('div', {
        staticClass: 'parallax__image-container'
      }, [
        h('img', {
          staticClass: 'parallax__image',
          style: this.styles,
          attrs: {
            src: this.src?this.src:"",
            srcset: this.srcset?this.srcset:"",
            heights: this.heights?this.heights:"",
            sizes: this.sizes?this.sizes:"",
          },
          ref: 'img'
        })
      ])

........
```

**Comment by ram-you** [comment]
FYI: W3C HTML 5 specifications for using **srcset**
http://w3c.github.io/html/semantics-embedded-content.html#element-attrdef-img-srcset

**Comment by johnleider** [comment]
Thank you!

**Comment by johnleider** [comment]
We are definitely going to support this! Will come with some cool new features as well.

**Comment by johnleider** [comment]
This will still be implemented but is just very low on triage.

**Comment by TGRBTRFLY** [comment]
Let me know when it gets implemented! :)

**Comment by KazakovKirill** [comment]
@TGRBTRFLY @ram-you Please check my commit #10913 — it working for me now

**Comment by KazakovKirill** [comment]
@TGRBTRFLY @ram-you, as temporary solution you can use this:

`mounted() {
    // I have only one parallax image
    const imgs = document.getElementsByClassName('v-parallax__image') 

    // Here is your srcset attribute value
    const srcset = require('~/assets/img/parallax.jpg').srcSet

    if (imgs && imgs.length > 0)
      Array.prototype.forEach.call(imgs, (img) => img.setAttribute('srcset', srcset))
  }`

**Comment by johnleider** [comment]
resolved in e72e78f3e9872c65589cfbdd197d23c214c8426e
